[micromega] Deprecate hopefully useless options and flags (#13781)
coq/coq: doc/sphinx/addendum/micromega.rst

.. flag:: Simplex

   .. deprecated:: 8.14

   This flag (set by default) instructs the decision procedures to

>    use the Simplex method for solving linear goals.
>    If it is not set, the decision procedures are using the *deprecated* Fourier elimination.

Reviewer: ```suggestion
   use the Simplex method for solving linear goals instead of the
   deprecated Fourier elimination.
```

Any special reason to emphasize `deprecated`?
